modules/rest-api/pages/rest-xdcr-adv-settings.adoc

```diff
@@ -217,34 +235,77 @@ If the call is successful, an object containing the settings for the specified r
   "collectionsMirroringMode": false,
   "collectionsOSOMode": true,
   "compressionType": "Auto",
+  "dcpEnablePurgeRollback": false,
   "desiredLatency": 50,
   "docBatchSizeKb": 2048,
   "failureRestartInterval": 10,
+  "filterBinary": false,
   "filterBypassExpiry": false,
   "filterBypassUncommittedTxn": false,
   "filterDeletion": false,
   "filterExpiration": false,
   "filterExpression": "",
-  "hlvPruningWindowSec": 259200,
   "jsFunctionTimeoutMs": 20000,
   "logLevel": "Info",
   "mergeFunctionMapping": {},
+  "mobile": "Off",
   "networkUsageLimit": 0,
   "optimisticReplicationThreshold": 256,
   "pauseRequested": false,
+  "preCheckCasDriftThresholdHours": 8760,
   "preReplicateVBMasterCheck": true,
   "priority": "High",
   "replicateCkptIntervalMin": 20,
+  "retryOnErrExceptAuthErrMaxWaitSec": 360,
   "retryOnRemoteAuthErr": true,
-  "retryOnRemoteAuthErrMaxWaitSec": 3600,
-  "sourceNozzlePerNode": 2,
+  "retryOnRemoteAuthErrMaxWaitSec": 360,
+  "skipReplSpecAutoGc": false,  "sourceNozzlePerNode": 2,
   "statsInterval": 1000,
   "targetNozzlePerNode": 2,
+  "targetTopologyLogFrequency": 1800,
   "type": "xmem",
   "workerBatchSize": 500
 }
 ----
 
+[#rest-api-conflict-logging-replication-tunables]
+*Example*: Retrieve the tunable settings for Conflict Logging in a replication.
+
+After you create a replication with conflict logging enabled, to view the replication settings, retrieve the replication's `settings_URI`. Here's an example.
+
+----
+curl -X GET -s -u Administrator:'password' http://localhost:8091/settings/replications/f6f3a51e988f305b16433237d02bec7d%2Fbucket3%2Fbucket3 | jq `.`
+----
+
+If the call is successful, an object is returned, which contains the replication settings for fine tuning the Conflict Logging feature:
+
+NOTE: For more information about fine tuning the replication settings in the Conflict Logging feature, see xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#tunable-replication-settings[Tunable Replication Settings].
```

Comment:
Don't say "fine tuning" -- we don't want customers to tune anything unless directed by Support.  I had suggested changing the heading "Tunable Replication Settings" to just "Conflict Log Settings" and "Configuring the conflictLogging Setting" to "Enabling and Configuring Conflict Logging" in the xdcr-conflict-logging-feature.adoc.


NOTE: For more information about the replication settings in the Conflict Logging feature, see xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#tunable-replication-settings[Conflict Log Settings] and xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#configure-conflictlogging-settings[Enabling and Configuring Conflict Logging].